MoneySplitter.Services/API/SessionApiService.cs

```diff
@@ -32,18 +32,18 @@ public SessionApiService(IApiUrlBuilder urlBuilder, IQueryApiService queryApiSer
 
             var dataUser = await _queryApiService.PostQueryAsync<DataUser, LoginModel>(loginModel, signInUri);
 
-            return _maper.DataUserToConvertUserModel(dataUser);
+            return _maper.ConvertDataUserToUserModel(dataUser);
```

Comment:
better rename this method to ConvertUserDataToUserModel

Reply:
fixed